dev/how-to/maintain/backup-and-restore/mydumper-lightning.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 使用 mydumper/tidb lightning 进行备份与恢复
+category: how-to
+aliases: ['/docs-cn/dev/how-to/maintain/backup-and-restore/mydumper-lightning']
+---
+
+# 备份与恢复
+
+本文档将详细介绍如何使用 `mydumper`/`tidb lightning` 对 TiDB 进行全量备份与恢复。增量备份与恢复可使用 [TiDB Binlog](/dev/reference/tidb-binlog/overview.md)。
```

Comment:
Either `tidb-lightning` (the exe name) or "TiDB Lightning" (the product name)

Reply:
+1